CHANGELOG.md

```diff
@@ -35,6 +35,24 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [v2.7.0] - 04-11-2023
+Version 2.7.0 brings many breaking changes with Oracle 2.0.
```

Comment:
How about something like:
Version 2.7.0 brings Oracle 2.0, which offers security evaluation for transactions running on business chains.